vcd/resource_vcd_org_saml.go

```diff
@@ -0,0 +1,210 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"log"
+	"os"
+)
+
+// resourceVcdOrgSaml handles Org SAML settings
+func resourceVcdOrgSaml() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourceVcdOrgSamlRead,
+		CreateContext: resourceVcdOrgSamlCreate,
+		UpdateContext: resourceVcdOrgSamlUpdate,
+		DeleteContext: resourceVcdOrgSamlDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdOrgSamlImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"org_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Organization ID",
+			},
+			"enabled": {
+				Type:        schema.TypeBool,
+				Required:    true,
+				Description: "Enable SAML authentication. When this option is set, authentication is deferred to the SAML identity provider",
+			},
+			"entity_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Your service provider entity ID. Once you set this field, it cannot be changed back to empty.",
+			},
+			"identity_provider_metadata_file": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The name of the file containing the metadata from the identity provider",
+			},
```

Comment:
My only thought here is - should we also accept text entry. That could be convenient if one uses another datasource to fetch the metadata 

https://github.com/vmware/terraform-provider-vcd/pull/1064/files#diff-6c14c3866e8fd9700b3a55a1aa50c84df75adda9644a846f297f4bc3af8f1b7aR61-R63